README.md

```diff
@@ -6,7 +6,7 @@ This repository contains PHIS users documentation.
 ## Background
 PHIS (Phenotyping Hybrid Information System) is developped for high-throughput plant phenotyping in close interaction to plant phenotyping communities.
 PHIS is part of SILEX collaborative meta-project developped at MISTEA joint research unit (INRA - SupAgro).
-More information on SILEX is availbale at [SILEX wiki](https://mulcyber.toulouse.inra.fr/plugins/mediawiki/wiki/silex/index.php/Accueil "SILEX wiki Main Page").
+More information on SILEX is available at [SILEX wiki](https://mulcyber.toulouse.inra.fr/plugins/mediawiki/wiki/silex/index.php/Accueil "SILEX wiki Main Page").
```

Comment:
SILEX wiki is not up to date and will be remove, may be refere to mistea website ou just remove this reference.

Reply:
corrected with the mention of MISTEA website in the last commit : a5253abb21ea205b3b5443b94b3f9c10b42aabbb